src/name_bazaar/shared/smart_contracts.cljs

```diff
@@ -33,4 +33,4 @@
   :address "0x2cfd99770f16cce3463fd376eab978632a501a58"},
  :auction-offering
  {:name "AuctionOffering",
-  :address "0xb02c63815fd3a90def33a4611070b2dd417a04f8"}})
\ No newline at end of file
+  :address "0xb02c63815fd3a90def33a4611070b2dd417a04f8"}})
```

Comment:
I think we should avoid commit noise in this file.

Reply:
Yeah, I really dislike that we cannot ignore this file in git though...